Source/WebKit/UIProcess/ios/WKContentViewInteraction.mm

```diff
@@ -1514,22 +1514,17 @@ - (CGRect)unobscuredContentRect
 
 
 #pragma mark - UITextAutoscrolling
+
```

Comment:
Should this whitespace be here?

Reply:
Yeah, this was an intentional drive-by style change: we usually put a newline after `#pragma mark`s.

Thanks for the review!